packages/rs-dpp/src/data_contract/data_contract.rs

```diff
@@ -111,6 +114,16 @@ impl DataContract {
         Ok(data_contract)
     }
 
+    pub fn from_json_object(json_value: JsonValue) -> Result<DataContract, ProtocolError> {
```

Comment:
why not `mut json_value: JsonValue` and no reassignment later?

Reply:
it was just the matter that we don't need to mark value as `mut` in upper body. Changed it back to `mut`